crates/viewer/re_view_spatial/src/ui.rs

```diff
@@ -154,6 +159,25 @@ impl SpatialViewState {
                 self.state_3d.set_spin(spin);
             }
         }
+
+        ui.horizontal(|ui| {
+            if let Some(tracked_entity) = self.state_3d.tracked_entity.clone() {
+                let (latest_at_query, _) =
+                    item_ui::guess_query_and_db_for_selected_entity(ctx, &tracked_entity);
```

Comment:
why not just read the current query from the `ctx` directly? `guess_query_and_db_for_selected_entity` does a lot more work and can produce the wrong result in this context